Setup.hs

```diff
@@ -1,5 +1,5 @@
 -- ------------------------------------------------------ --
--- Copyright © 2015-2018 Lars Kuhtz <lakuhtz@gmail.com>
+-- Copyright © 2015-2019 Lars Kuhtz <lakuhtz@gmail.com>
```

Comment:
I am not a copyright expert, but I think we may add a new line for 2019 here with the copyright of the author of this PR. Similar with other files that have code changes.

Reply:
I think that meaning would be that the copyright for 2019 only covers the change that got added in that year. Not 100% sure though.